### PR TITLE
OSDOCS4041: Support Windows Containers on GCP -- WMCO 7.0.0

### PR DESCRIPTION
**Pull request by mburke5678**
https://issues.redhat.com/browse/OSDOCS-4041

Previews:
[Creating a Windows MachineSet object on GCP](http://file.rdu.redhat.com/mburke/winc-add-gcp/windows_containers/creating_windows_machinesets/creating-windows-machineset-gcp.html) Intro and the [Sample YAML for a Windows MachineSet object on GCP](http://file.rdu.redhat.com/mburke/winc-add-gcp/windows_containers/creating_windows_machinesets/creating-windows-machineset-gcp.html#windows-machineset-gcp_creating-windows-machineset-gcp)
 module. The other modules are existing and repreated in all _Creating a Windows MachineSet_ assemblies.

[WMCO 7.0.0 supported platforms and Windows Server versions](http://file.rdu.redhat.com/mburke/winc-add-gcp/windows_containers/windows-containers-release-notes-6-x.html#wmco-prerequisites-supported-6.0.0_windows-containers-release-notes) -- Added GCP to table

### CODE REVIEW

**Comment by mburke5678** [comment]
@sebsoto Are OK to review my PR at this time? I know we might have to revisit pending the outcome of https://issues.redhat.com/browse/OCPCLOUD-1533

**Comment by mburke5678** [comment]
@sebsoto Are OK to review my PR at this time? I know we might have to revisit pending the outcome of https://issues.redhat.com/browse/OCPCLOUD-1533

**Comment by mburke5678** [comment]
@sebsoto @jrvaldes PTAL

**Review by elmiko — COMMENTED** [review]
/lgtm

File: modules/windows-machineset-gcp.adoc [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+<1> Specify the infrastructure ID that is based on the cluster ID that you set when you provisioned the cluster. You can obtain the infrastructure ID by running the following command:
++
+[source,terminal]
+----
+$ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
+----
+<2> Specify the infrastructure ID, worker label, and zone suffix (such as `a`).
+<3> Configure the machine set as a Windows machine.
+<4> Configure the Windows node as a compute machine.
+<5> Specify the full path to an image of a supported version of Windows.
```
Comment (jrvaldes) [inline modules/windows-machineset-gcp.adoc:76]:
```suggestion
<5> Specify the full path to an image of a supported version of Windows Server. 
```

**Review by jrvaldes — COMMENTED** [review]
LGTM, just a minor suggestion.

@mburke5678 thanks for working on this.

File: modules/windows-machineset-gcp.adoc [hunk 2]
```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * windows_containers/creating_windows_machinesets/creating-windows-machineset-gcp.adoc
+
+[id="windows-machineset-gcp_{context}"]
+= Sample YAML for a Windows MachineSet object on GCP
+
+This sample YAML defines a Windows `MachineSet` object running on Google Cloud Platform (GCP) that the Windows Machine Config Operator (WMCO) can react upon.
```
Comment (jldohmann) [inline modules/windows-machineset-gcp.adoc:8]:
```suggestion
This sample YAML file defines a Windows `MachineSet` object running on Google Cloud Platform (GCP) that the Windows Machine Config Operator (WMCO) can react upon.
```

File: windows_containers/creating_windows_machinesets/creating-windows-machineset-gcp.adoc [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+* You installed the Windows Machine Config Operator (WMCO) using Operator Lifecycle Manager (OLM).
+* You are using a supported Windows Server as the operating system image. 
+
+include::modules/machine-api-overview.adoc[leveloffset=+1]
+include::modules/windows-machineset-gcp.adoc[leveloffset=+1]
+include::modules/machineset-creating.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+== Additional resources
```
Comment (jldohmann) [inline windows_containers/creating_windows_machinesets/creating-windows-machineset-gcp.adoc:22]: This [needs an ID](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies), something like:
```suggestion
[id="additional-resources_creating-windows-machineset-gcp"]
== Additional resources
```

File: windows_containers/creating_windows_machinesets/creating-windows-machineset-gcp.adoc [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+* For more information on managing machine sets, see xref:../../machine_management/index.adoc#overview-of-machine-management[Overview of machine management].
```
Comment (jldohmann) [inline windows_containers/creating_windows_machinesets/creating-windows-machineset-gcp.adoc:24]: This link in the preview gives me a 404 error

**Review by jldohmann — COMMENTED** [review]
Mostly looking good! Feedback below

File: windows_containers/creating_windows_machinesets/creating-windows-machineset-gcp.adoc (hunk 4 above)
Comment (mburke5678) [reply windows_containers/creating_windows_machinesets/creating-windows-machineset-gcp.adoc:24]: @jldohmann All apologies. I sent only the Windows Containers docs to file.rdu.redhat to save space. The Netlify preview works: https://49719--docspreview.netlify.app/openshift-enterprise/latest/windows_containers/creating_windows_machinesets/creating-windows-machineset-gcp.html#additional-resources

Comment (jldohmann) [reply windows_containers/creating_windows_machinesets/creating-windows-machineset-gcp.adoc:24]: No worries, thanks for directing me to the netlify preview to test the link!

**Comment by mburke5678** [comment]
DO NOT MERGE until https://issues.redhat.com/browse/WINC-812 is merged

**Comment by bergerhoffer** [comment]
The `branch/enterprise-4.13` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.12. And any PR going into `main` must also target the latest version branch (enterprise-4.13).

If the update in your PR does NOT apply to version 4.13 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Pull request merged**